articles/web-dev's-guide-SEO/index.md

```diff
@@ -0,0 +1,85 @@
+# Developer's Guide to Search Engine Optimization
+
+Search engine optimization (SEO) is an incredibly important aspect of designing websites. SEO is usually tackled by the marketing department in a given company. But, developers have an equal role to play in ensuring the website is easily found on the search engines. When a URL is shared, how can we ensure that the experience with the URL is enhanced? For example, YouTube or Facebook links can be opened within the app with the help of the [picture-in-picture](https://support.google.com/youtube/answer/7552722?co=GENIE.Platform%3DAndroid&hl=en) feature. When we place an order on Amazon, the order details and delivery estimate is available to our Google assistant. Some links provide a vanilla experience with nice imagery and graphics, smooth transitions between apps, for example, LinkedIn URLs.
+
+Search engine optimization is the art of standing out amongst other websites. This art is pioneered by creating an experience that the user seeks repetitively, thereby increasing the presence on the internet. We will look at some of the key considerations a developer should keep in mind while designing the website. The techniques aim at simplifying the job of the millions of [crawlers that search and index the internet regularly](https://www.sciencedaily.com/terms/web_crawler.htm). We will look at the various aspects of SEO, and consider the important concepts of Metadata. Towards the end of the article, you will have a solid grasp of some good SEO techniques. 
+
+## Traditional SEO
+
+Let us look at some of the ways that are used to improve SEO. These are factors responsible for ensuring crawlers can do their job easily. Earlier, the HTML meta tag was used to fill in a set of important keywords. Severe exploitation of the same has resulted in search engines disregarding the keywords. Crawlers are bots that go through pages and find the content present there and list the various links the website directs to. By doing so, crawlers map the internet and enhance the performance of search engines. 
+
+At the same time, the content plays a major role. The amount of time a user spends and the monetization capacity are tracked regularly by the crawlers. Finally, target the niche audience matters. Providing unique content that is designed, delivered, and branded has a higher chance of being ranked higher. The [investment on User Interface and User Experience(UI/UX)](https://neilpatel.com/blog/googlebot-optimization/) returns a larger time spent on the site and more number of clicks resulting in better SEO. 
```

Comment:
the target niche makes more sense.

There should be a space between Experience and the bracket. 

You should replace more with a larger.